packages/target-nodejs-sdk/src/target.js

```diff
@@ -128,6 +132,13 @@ export function executeDelivery(options, decisioningEngine) {
         Messages.RESPONSE_RECEIVED,
         JSON.stringify(response, null, 2)
       );
+
+      if (deliveryMethod.decisioningMethod === DECISIONING_METHOD.SERVER_SIDE) {
+        telemetryProvider.addEntry(deliveryRequest, {
+          execution: timingTool.timeEnd(TIMING_EXECUTE_DELIVERY)
```

Comment:
Let's move `timingTool.timeEnd(TIMING_EXECUTE_DELIVERY)` above the logger.debug statement so we measure as soon as we can.